### PR TITLE
Method for test orders API 

### PR DESCRIPTION
**Pull request by d-m-ivanov**

### CODE REVIEW

File: exchange/binanceclient.py [hunk 1]
```diff
@@ -42,29 +46,43 @@ def new_order_market(self, side: str, quantity=None, quote_order_qty=None,
                                  within a certain number of milliseconds or be rejected by the server.
         """
```
Comment (Atem1os) [inline exchange/binanceclient.py:47]: maybe change type_ to order_type

File: exchange/binanceclient.py [hunk 2]
```diff
@@ -42,29 +46,43 @@ def new_order_market(self, side: str, quantity=None, quote_order_qty=None,
...
+            params["stopPrice"] = stop_price
+        params["recvWindow"] = recv_window
+        params["timestamp"] = self.get_now_timestamp()
+        total_params = "&".join([key + "=" + str(value) for key, value in params.items()])
         params["signature"] = self._get_signature(total_params)
+        resp = requests.post("https://api.binance.com/api/v3/order/test", headers=headers, params=params)
+        return resp.json()
+
+    def try_order(self, side: str, type_="MARKET", time_in_force='GTC',
+                  quantity=None, quote_order_qty=None, price=None,
```
Comment (Atem1os) [inline exchange/binanceclient.py:68]: maybe send_test_order() better?

File: exchange/binanceclient.py [hunk 3]
```diff
@@ -73,23 +91,21 @@ def cancel_order(self, order_id, recv_window=5000):
...
         headers = {'X-MBX-APIKEY': self.api}
         params = {"symbol": self.pair,
                   "recvWindow": recv_window,
-                  "timestamp": 0,
+                  "timestamp": self.get_now_timestamp(),
                   "signature": None}
         total_params = "&".join([key + "=" + str(value) for key, value in params.items() if key != "signature"])
         params["signature"] = self._get_signature(total_params)
-        params["timestamp"] = self.get_now_timestamp()
         resp = requests.delete("https://api.binance.comapi/v3/openOrders", headers=headers, params=params)
```
Comment (Atem1os) [inline exchange/binanceclient.py:109]: Maybe change get_pandas_df() to candlesticks_to_pandas() or smth different?
Also, typo in check_interval?

File: exchange/binanceclient.py [hunk 4]
```diff
@@ -23,15 +23,19 @@ def __init__(self, base_asset, quote_asset, api_key='', secret_key=''):
         self.pair = self._get_pair()
         self._chek_pair()
```
Comment (Atem1os) [inline exchange/binanceclient.py:25]: rename supp_script to smth like utils.py or helper.py (or binance_client_helper.py, assuming that there's going to be other helpers)?

**Review by Atem1os — CHANGES_REQUESTED** [review]
Overall good, didn't test all methods, some minor style caveats.

File: exchange/binanceclient.py (hunk 3 above)
Comment (d-m-ivanov) [reply exchange/binanceclient.py:109]: May be you can specify what kind of typo do you mean? I can't see any typo in check_interval.

Comment (Atem1os) [reply exchange/binanceclient.py:109]: изображение

**Review by Atem1os — APPROVED** [review]

**Review by Atem1os — APPROVED** [review]

**Review by Atem1os — APPROVED** [review]

**Review by Atem1os — APPROVED** [review]

**Review by Atem1os — APPROVED** [review]

**Pull request merged**